WalletWasabi/WabiSabi/Backend/Rounds/CoinJoinStorage/CoinJoinTransactionArchiver.cs

```diff
@@ -58,6 +105,7 @@ public async Task<string> StoreJsonAsync(Transaction transaction, DateTimeOffset
 	/// <param name="RawTransaction">Raw bitcoin transaction in hexadecimal representation.</param>
 	public record TransactionInfo(
 		[property: JsonPropertyName("createdMs")] long Created,
+		[property: JsonPropertyName("feeRate")] decimal FeeRate,
```

Comment:
We can document it as: `/// <param name="FeeRate">Fee rate of the transaction.</param>`

It looks like it is what it means.